source/views/building-hours/list.tsx

```diff
@@ -62,7 +62,13 @@ function groupBuildings(
 export function BuildingHoursView(): JSX.Element {
 	let navigation = useNavigation()
 
-	let {data: buildings, isPending, isInitial, reload} = useBuildingHours()
+	let {
+		data: {data: buildings} = [],
```

Comment:
I've introduced a type error as a result of this destructuring –– tracked in commit https://github.com/StoDevX/AAO-React-Native/pull/6021/commits/5d6d4aed8300dd2022ce79f8202479aea386f22a.

Reply:
We'll need to edit the return type of useBuildingHours to say `{data: BuildingHour[]}`, I think